docker-compose.yml

```diff
@@ -180,7 +180,7 @@ services:
       CONNECT_PLUGIN_PATH: "/usr/share/java,/connect-plugins"
       CONNECT_LOG4J_ROOT_LOGLEVEL: INFO
       CONNECT_LOG4J_LOGGERS: org.reflections=ERROR
-      CLASSPATH: /usr/share/java/monitoring-interceptors/monitoring-interceptors-5.4.0-SNAPSHOT.jar
+      CLASSPATH: /usr/share/java/monitoring-interceptors/monitoring-interceptors-5.4.x-latest.jar
```

Comment:
JARs are -SNAPSHOT, correct?

Reply:
jar should be snapshot, we do not use latest for jars.

Reply:
Right, you've changed it to `-latest` here

Reply:
my bad, fixed this.